email_sender/email_sender.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var fs = require('fs');
+var optionsBrowser16 = require('../test/itg/1.6/common.webdriverio');
+var optionsBrowser17 = require('../test/itg/1.7/common.webdriverio');
+
+// create reusable transporter object using the default SMTP transport
+var nodeMailer = require('nodemailer');
+var dateFormat = require('dateformat');
+
+// get travis env variable
+var senderEmail = process.env.SENDER_EMAIL;
+var senderPassword = process.env.SENDER_PASSWORD;
+var recipientEmail = process.env.RECIPIENT_EMAIL;
+
+var prestaVersion = new Array();
+prestaVersion = [1.6, 1.7];
```

Comment:
We should not have these hard-coded versions. We can find the current version in the travis environment, and then can be send to this script as a parameter.